Source/WebCore/html/HTMLFormElement.cpp

```diff
@@ -261,22 +261,17 @@ bool HTMLFormElement::validateInteractively()
 
     Ref<HTMLFormElement> protectedThis(*this);
 
+    Vector<RefPtr<ValidatedFormListedElement>> unfocusableControls;
```

Comment:
Looks like this variable is never used?

Reply:
Right, [c7f0a67](https://github.com/WebKit/WebKit/pull/4988/commits/c7f0a6763aed67f4cb58f291ac4dcd4aaa800485).